datacache/common.py

```diff
@@ -65,7 +65,7 @@ def build_local_filename(download_url=None, filename=None, decompress=False):
 
     # if no filename provided, use the original filename on the server
     if not filename:
-        digest = hashlib.md5(download_url).hexdigest()
+        digest = hashlib.md5(download_url.encode('utf-8')).hexdigest()
```

Comment:
Can you comment this or at least explain it to me? Is it so Python 2.7.x and 3.x produce the same hexdigests?